var/spack/repos/builtin/packages/fujitsu-ssl2/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class FujitsuSsl2(Package):
+    """Fujitsu SSL2 (Scientific Subroutine Library II) is Math library
+    for Fujitsu compiler.
+    Fujitsu SSL2 implementation only for Fujitsu compiler.
+    Fujitsu SSL2 provides the function of blas, lapack and scalapack.
+    """
+
+    homepage = "https://www.fujitsu.com/us/"
+
+    conflicts("%arm")
+    conflicts("%cce")
+    conflicts("%apple-clang")
+    conflicts("%clang")
+    conflicts("%gcc")
+    conflicts("%intel")
+    conflicts("%nag")
+    conflicts("%pgi")
+    conflicts("%xl")
+    conflicts("%xl_r")
+
+    provides("blas")
+    provides("lapack")
+    provides("scalapack")
+
+    def install(self, spec, prefix):
+        raise InstallError(
+            "Fujitsu SSL2 is not installable; it is vendor supplied")
```

Comment:
I suggest providing a message here that is clearer to the person who wants to use your software.  For example (from the `lustre` package) replacing the message here with:


```suggestion
            "Fujitsu SSL2 is not installable; it is vendor supplied"
            self.spec.format('{name} is not installable, you need to specify '
                             'it as an external package in packages.yaml')))
```